### DISCUSSION TITLE
No robots.txt on new install via docker image

### DISCUSSION THREAD

**Discussion by MartinSchorel** [requarks/wiki#4788] [Help / Questions] [answered]
All,

Sorry for the newbie question, but there is a configuration setting in the GUI to detail the robots.txt content, however when i request the xx.yy.zz/robots.txt there seems to be no content?

Any ideas?

**Comment by NGPixel** [comment, marked as answer]
An empty robots.txt is actually the recommended default. Rules are only added when they are different from the defaults.

**Reply by MartinSchorel** [reply]
Thanks @NGPixel for pointing this out.. as mentioned, Newbie to wiki.js but already impressed!
Fiddeling around with some other (think docker related things) but boy, does this wiki fly!
Can't wait for the 3.0 release...
Let me also investigate the backing uptions you have!
